c/validation/adbc_validation.cc

```diff
@@ -2112,7 +2112,8 @@ void StatementTest::TestSqlIngestErrors() {
                                          {"coltwo", NANOARROW_TYPE_INT64}}),
               IsOkErrno());
   ASSERT_THAT(
-      (MakeBatch<int64_t, int64_t>(&schema.value, &array.value, &na_error, {}, {})),
+      (MakeBatch<int64_t, int64_t>(&schema.value, &array.value, &na_error,
```

Comment:
Looks like postgres COPY only raises if there is data

```
postgres=# COPY (SELECT int64s, int64s FROM bulk_ingest WHERE 1=0) TO '/tmp/pgout.data' WITH (FORMAT BINARY);
postgres=# COPY bulk_ingest FROM '/tmp/pgout.data' WITH (FORMAT BINARY);
COPY 0
postgres=# COPY (SELECT int64s, int64s FROM bulk_ingest) TO '/tmp/pgout.data' WITH (FORMAT BINARY);
postgres=# COPY bulk_ingest FROM '/tmp/pgout.data' WITH (FORMAT BINARY);
ERROR:  row field count is 2, expected 1
CONTEXT:  COPY bulk_ingest, line 1
```

Given the format itself sends the number of columns expected row-by-row; in the empty result case no such communication occurs